ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -968,6 +1144,11 @@ export const RuntimePanel = fp.flow(
   const standardDiskValidator = {
     selectedDiskSize: diskSizeValidatorWithMessage('standard')
   };
+
+  const standardPdValidator = {
```

Comment:
Per above comment on making these the same variable - it would eliminate this duplication

Reply:
@YintongMa I think the suggestion is just to use standardDisk* for everything, instead of creating a new one.